### PR TITLE
Fix errcheck issues in server/channels/utils/fileutils/fileutils_test.go

### PR DESCRIPTION
**Pull request by shraddha761**
#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/28753
JIRA: [MM-61081](https://mattermost.atlassian.net/browse/MM-61081)

#### Release Note

```release-note

```

[MM-61081]: https://mattermost.atlassian.net/browse/MM-61081?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: server/channels/utils/fileutils/fileutils_test.go [hunk 1]
```diff
@@ -1,6 +1,5 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
-
```
Comment (hanzei) [inline server/channels/utils/fileutils/fileutils_test.go:3]: Please remove the unrelated change

File: server/channels/utils/fileutils/fileutils_test.go [hunk 2]
```diff
@@ -27,7 +26,10 @@ func TestFindFile(t *testing.T) {
 		//         tmpDir5/
 		tmpDir1, err := os.MkdirTemp("", "")
 		require.NoError(t, err)
-		defer os.RemoveAll(tmpDir1)
+		defer func() {
+			err := os.RemoveAll(tmpDir1)
+			require.NoError(t, err)
+		}()
```
Comment (hanzei) [inline server/channels/utils/fileutils/fileutils_test.go:32]: No need for a new `err` variable. You can just use the existing one
```suggestion
		defer func() {
			err = os.RemoveAll(tmpDir1)
			require.NoError(t, err)
		}()
```

File: server/channels/utils/fileutils/fileutils_test.go [hunk 3]
```diff
@@ -46,16 +48,16 @@ func TestFindFile(t *testing.T) {
 
 		type testCase struct {
 			Description string
-			Cwd         *string
-			FileName    string
-			Expected    string
+			Cwd        *string
+			FileName   string
+			Expected   string
```
Comment (hanzei) [inline server/channels/utils/fileutils/fileutils_test.go:53]: Unrelated change

File: server/channels/utils/fileutils/fileutils_test.go [hunk 4]
```diff
@@ -112,12 +114,17 @@ func TestFindFile(t *testing.T) {
...
-					defer os.Chdir(prevDir)
-					os.Chdir(*testCase.Cwd)
+					
+					err = os.Chdir(*testCase.Cwd)
+					require.NoError(t, err)
+					
+					defer func() {
+						err := os.Chdir(prevDir)
+						require.NoError(t, err)
+					}()
```
Comment (hanzei) [inline server/channels/utils/fileutils/fileutils_test.go:124]: Same here
```suggestion
					defer func() {
						err = os.Chdir(prevDir)
						require.NoError(t, err)
					}()
```

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the PR! 


Can you please remove the unrelated change? Thanks!

File: server/channels/utils/fileutils/fileutils_test.go [hunk 5]
```diff
@@ -52,10 +55,10 @@ func TestFindFile(t *testing.T) {
 		}
 
 		testCases := []testCase{}
-
```
Comment (hanzei) [inline server/channels/utils/fileutils/fileutils_test.go:55]: Nit: Unrelated change

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Review by fmartingr — APPROVED** [review]
👍  Thank you!

File: server/channels/utils/fileutils/fileutils_test.go [hunk 6]
```diff
@@ -112,10 +115,15 @@ func TestFindFile(t *testing.T) {
 				if testCase.Cwd != nil {
 					prevDir, err := os.Getwd()
 					require.NoError(t, err)
-					defer os.Chdir(prevDir)
-					os.Chdir(*testCase.Cwd)
+
```
Comment (hanzei) [inline server/channels/utils/fileutils/fileutils_test.go:118]: The file is not `gofmt`ed. Can you please fix that?

**Comment by hanzei** [comment]
Thanks @shraddha761 !

**Pull request merged**